src/pin-field/PinField.stories.ts

```diff
@@ -108,4 +108,4 @@ export const Prefix = PrefixStory<PinField, BaseArgs>('omni-pin-field');
 
 export const Suffix = SuffixStory<PinField, BaseArgs>('omni-pin-field');
 
```

Comment:
1234? Pin numbers are normally 4 digits min in length.

Reply:
updated to 4 digits